v2/support/secure_settings_sample_app/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Bag End Catering Helper",
+  "author": {
+    "name": "Zendesk",
+    "email": "support@zendesk.com",
+    "url": "https://support.zendesk.com"
+  },
+  "defaultLocale": "en",
+  "private": true,
+  "location": {
+    "support": {
+      "ticket_sidebar": "assets/iframe.html"
+    }
+  },
+  "parameters": [
```

Comment:
We should also include a [`domainWhitelis`](https://developer.zendesk.com/apps/docs/developer-guide/using_sdk#using-secure-settings).

Reply:
Ooops.  I'll get to it.